bluej/src/main/java/bluej/parser/EditorParser.java

```diff
@@ -1085,6 +1088,10 @@ else if (gotImplements) {
                 implementedTypes.add(supert);
             }
         }
+        else if (gotPermits)
+        {
+            // Currently, do nothing with this
+        }
```

Comment:
Just to make sure that I got this correctly: the comment here indicate that visually, in the class diagramme, we do nothing specific for the classes that are permitted, correct?